### PR TITLE
177 Selected Answer Set & Possible Solution to a Question not saved correctly

### PR DESCRIPTION
**Pull request by iTitus**
Partially ~~fixes~~ #177 

This fixes question answering to accept multiple-/single-choice answers.

### CODE REVIEW

**Comment by dpkass** [comment]
You keep using the term solution-ids in the code. Do you mean the right answer by solution, or possible solutions. 

If you mean possible solutions, then please update as it is very misleading. I was very confused.

File: src/clj/controllers/question/question_controller.clj [hunk 1]
```diff
@@ -76,8 +76,7 @@
                                           achivable-points
                                           type
                                           possible-solutions
-                                          single-choice-solution
-                                          multiple-choice-solution
+                                          (or single-choice-solution multiple-choice-solution)
```
Comment (iTitus) [inline src/clj/controllers/question/question_controller.clj:79]: This was a little bit overeager, I wanted to fix this in the next PR.
But thanks ;)

**Comment by iTitus** [comment]
> You keep using the term solution-ids in the code. Do you mean the right answer by solution, or possible solutions.
> 
> If you mean possible solutions, then please update as it is very misleading. I was very confused.

A solution is an entity in the database, it just contains a statement and an id.
I will add it to the glossary.

**Comment by dpkass** [comment]
I've added some fixes. It was not possible to create a question, because of an old validation error. It is still not possible to create choice questions, but that should be a different ticket.

**Review by dpkass — APPROVED** [review]

**Comment by dpkass** [comment]
Those are actually possible solutions. A solution is the right answer to a question..

Mit freundlichen Grüßen


Taha El Amine Kassabi
________________________________

**Pull request merged**